engine/src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -84,6 +71,12 @@ trait Backend {
 
   def actorSystem: ActorSystem
 
+  def backendConfigEntry: BackendConfigurationEntry
```

Comment:
There are still some engine dependencies here, but it looks like we're getting closer to being able to move this.